cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/HttpRequestBuilderImpl.java

```diff
@@ -217,8 +220,26 @@ public HttpRequestExecutor withMediaType(String type) {
         }
     }
 
+    public final String download(String urlString) throws IOException {
```

Comment:
I'm not sure this is the right place for this method.   Also, it seems like it could be static - is there something non-static happening that I'm not seeing?

Reply:
I don't see anything non-static happening. I'll change it.

Reply:
I could put it in the executor

Reply:
I would think that this download method would be very similar to ClobController would be closer to what the ClobController.retrieveClob  method does except that you'd return String.   You probably want to have a withMediaType(...) call that sets text/plain and you probably want to have a try-with-resources block around a HttpRequestResponse.   Once you have that HttpRequestResponse I suspect you'd get your clob out by doing some sort of readFully on response.getStream().

Those are just my first ideas.  I don't know if HttpRequestResponse.getStream is going to do the right thing, just hopeful.

Reply:
Done. Moved function into new OkHttpDownloader class.